telegram-bot-api/StatsJson.h

```diff
@@ -198,12 +198,12 @@ class JsonStatsBotStats : public td::Jsonable {
 
 class JsonStatsBotAdvanced : public JsonStatsBot {
  public:
-  explicit JsonStatsBotAdvanced(const std::pair<td::int64, td::uint64> score_id_pair,
+  explicit JsonStatsBotAdvanced(std::pair<td::int64, td::uint64> score_id_pair,
                                 const ServerBotInfo bot,
                                 const td::vector<ServerBotStat> stats,
                                 const bool hide_sensible_data,
                                 const double now)
-      : JsonStatsBot(score_id_pair), bot_(bot), stats_(stats), hide_sensible_data_(hide_sensible_data), now_(now) {
+      : JsonStatsBot(std::move(score_id_pair)), bot_(bot), stats_(stats), hide_sensible_data_(hide_sensible_data), now_(now) {
```

Comment:
also move bot and stats as you are already moving them from clientmanager

Reply:
Ah, okey. Sure will do.